start_esp/nginx-auto.conf.template

```diff
@@ -48,10 +48,7 @@ http {
 % endif
 % endfor
 
-  set_real_ip_from  0.0.0.0/0;
-  set_real_ip_from  0::/0;
-  real_ip_header    X-Forwarded-For;
-  real_ip_recursive on;
```

Comment:
let us go with implementing it ourrself.  it has two benefits
1) not need to change nginx config
2) it is more flexible, we can do any ways we wanted.